app/move/controllers/assign/base.js

```diff
@@ -13,6 +15,21 @@ class CreateAllocationBaseController extends CreateBaseController {
     next()
   }
 
+  setAllocationBlurb(req, res, next) {
```

Comment:
I don't think this is the most efficient place for this method. This is run on every step, but in reality it will only be needed on the step that uses this field.

It would be better in the two controllers related to people. But as discussed on slack I think a better approach would be to use more generic hint text like we have elsewhere and to provide the dynamic information via the right hand panel.

Reply:
Indeed. I have run the matter past @Andy-Millar , awaiting feedback.